environs/bootstrap/bootstrap.go

```diff
@@ -187,6 +187,13 @@ func bootstrapCAAS(
 	args BootstrapParams,
 	bootstrapParams environs.BootstrapParams,
 ) error {
+	if args.BuildAgent {
```

Comment:
I think there's other bootstrap params like Series etc that are also not supported?

Reply:
yes, added a few more.